.github/workflows/auto-build.yml

```diff
@@ -0,0 +1,252 @@
+name: Auto build image
+on:
+  issue_comment:
+    types:
+      - created
+jobs:
+  issue_comment:
+    name: Auto build image
+    if: startswith(github.event.comment.body, '/imagebuild')
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+
+      - name: Check outputs
+        run: |
+          commentbody="${{github.event.comment.body}}"
+          version=`echo "$commentbody"|cut -d" " -f2`
+          echo $version
+          arch=`echo "$commentbody"|cut -d" " -f3`
+          echo $arch
+          wget http://sealer.oss-cn-beijing.aliyuncs.com/auto-build/k8sv1.20.14.tar.gz
```

Comment:
I think you can from a BaseImage witch not contains kubernetes bin files, only has docker.

Reply:
Yes, this file does not contain kubernetes binaries